requirements-py38.txt

```diff
@@ -2,14 +2,14 @@
 # This file is autogenerated by pip-compile with python 3.8
 # To update, run:
 #
-#    pip-compile --generate-hashes --output-file=requirements-py38.txt
+#    pip-compile --allow-unsafe --generate-hashes --output-file=requirements-py38.txt
```

Comment:
we shouldn't allow unsafe

Reply:
I agree, but it's not building.. Still didn't find a way to include `setuptools` without it, unless I do it manually....

Reply:
[PTAL](https://app.travis-ci.com/github/release-engineering/iib/jobs/554399473) on Travis without `--allow-unsafe`, it is, without `setuptools`

Reply:
We can try to pin setuptools in iib-playbooks